tests/helpers/fake-redis-client.js

```diff
@@ -11,10 +12,12 @@ module.exports = CoreObject.extend({
   set: function() { },
   del: function() { },
   zadd: function(key, score , tag) {
-    this.recentRevisions.push(key + ':' + tag);
+    assert(key.match(/:revisions$/));
```

Comment:
yeah, as mentioned, I agree..
it spans across a bunch of tests, but we already reopen the redis client in some tests anyway so it should do no harm..

will ping you shortly with an update